src/Banking/Statement.php

```diff
@@ -23,6 +23,12 @@ class Statement
     /** @var Operation[] */
     private $operations = [];
 
+    public function __construct()
+    {
+        $this->oldBalance = null;
+        $this->newBalance = null;
```

Comment:
Could you move the `$operations = [];` in the constructor to be more consistent? As a general rule we will do all the property initialisations to the `__construct` method if you want to use the Rich Domain Model pattern now.

Reply:
good catch :)